src/helpers/productStringGenerator.js

```diff
@@ -0,0 +1,6 @@
+function productStringGenerator(tv) {
+    const productString = tv.brand + ' ' + tv.type + ' - ' + tv.name;
```

Comment:
Ik zie dat je soms de backticks gebruikt, en soms losse strings aan elkaar koppelt met plustekens. Ik denk dat het netter is om hierin consequent te blijven (en ik denk dat de backticks dan de voorkeur hebben).